src/Compatibility/Microsoft.DotNet.ApiCompatibility/Resources.resx

```diff
@@ -117,9 +117,27 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AssemblyNameDoesNotExist" xml:space="preserve">
+    <value>Assembly with name '{0}' does not exist at {1}.</value>
+  </data>
+  <data name="AssemblyNameDoesNotMatch" xml:space="preserve">
+    <value>{2} assembly name '{0}' does not match with {3} assembly name '{1}'.</value>
+  </data>
+  <data name="AssemblyPublicKeyTokenDoesNotMatch" xml:space="preserve">
+    <value>{2} assembly token '{0}' does not match with {3} '{1}'.</value>
+  </data>
   <data name="AssemblySearchDirectoryDoesNotExist" xml:space="preserve">
     <value>Provided assembly search directory '{0}' does not exist.</value>
   </data>
+  <data name="AssembyCultureDoesNotMatch" xml:space="preserve">
+    <value>{2} assembly culture '{0}' does not match with {3} assembly culture '{1}'.</value>
+  </data>
+  <data name="AssembyVersionDoesNotMatch" xml:space="preserve">
+    <value>{3} assembly version '{1}' should be equal to  {2} version '{0}'.</value>
+  </data>
+  <data name="AssembyVersionIsNotCompatible" xml:space="preserve">
+    <value>{3} assembly version '{1}' should be not higher than {2} version '{0}'.</value>
```

Comment:
Assembly version can be equal or higher when in strict mode, this error string still seems wrong.